### PR TITLE
Check PROGRESSING column to ensure all operators get ready

### PR DESCRIPTION
**Pull request by jianlinliu**

### CODE REVIEW

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-4.15-upgrade-from-stable-4.14-azure-ipi-workers-rhel8-f28

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-4.15-upgrade-from-stable-4.14-azure-ipi-workers-rhel8-f28

**Comment by jianlinliu** [comment]
cc @jiajliu @shellyyang1989 to review, per https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-4.15-upgrade-from-stable-4.14-azure-ipi-workers-rhel8-f28/1725669110039187456 , the change take effect.

File: ci-operator/step-registry/cucushift/upgrade/healthcheck/cucushift-upgrade-healthcheck-commands.sh [hunk 1]
```diff
@@ -83,6 +77,18 @@ function check_clusteroperators() {
...
     fi
 
+    echo "Make sure every operator's PROGRESSING column is False"
+    if progressing_operator=$(${OC} get clusteroperator | awk '$4 == "True"' | grep "True"); then
+        echo >&2 "Some operator's PROGRESSING is True"
+        echo >&2 "$progressing_operator"
+        (( tmp_ret += 1 ))
+    fi
+    if ${OC} get clusteroperator -o json | jq '.items[].status.conditions[] | select(.type == "Progressing") | .status' | grep -iv "False"; then
+        echo >&2 "Some operators are unavailable, pls run 'oc get clusteroperator -o json' to check"
```
Comment (shellyyang1989) [inline ci-operator/step-registry/cucushift/upgrade/healthcheck/cucushift-upgrade-healthcheck-commands.sh:87]: nit: unavailable -> progressing

Comment (jianlinliu) [reply ci-operator/step-registry/cucushift/upgrade/healthcheck/cucushift-upgrade-healthcheck-commands.sh]: fixed.

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-4.15-upgrade-from-stable-4.10-gcp-ipi-f14

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-arm64-nightly-4.14-upgrade-from-eus-4.12-azure-ipi-f28

**Comment by shellyyang1989** [comment]
Thanks for the universal fix.

/lgtm

**Comment by jianlinliu** [comment]
cc @liangxia to review

**Pull request merged**